src/main/java/nextstep/optional/User.java

```diff
@@ -33,6 +35,15 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
+        Optional<User> optional = Optional.ofNullable(user);
+        if (optional.isPresent()) {
+            optional.map(user1 -> user1.getAge() != null);
+            return optional
+                    .filter(optionalUser -> optionalUser.getAge() != null
+                            && optionalUser.getAge() >= 30
+                            && optionalUser.getAge() <= 45)
```

Comment:
```suggestion
                            && optionalUser.범위검증()
```
객체에게 범위 검증하라고 시켜봐도 좋을것 같아요!